src/__tests__/hover.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react'
+import {render, screen} from '@testing-library/react'
+import userEvent from '..'
+import {setup} from './helpers/utils'
+
+test('hover', () => {
+  const {element, getEventCalls} = setup(<button />)
+
+  userEvent.hover(element)
+  expect(getEventCalls()).toMatchInlineSnapshot(`
+    mouseenter: Left (0)
+    mouseover: Left (0)
+    mousemove: Left (0)
+    mouseout: Left (0)
+    mouseleave: Left (0)
```

Comment:
I'm guessing a lot of people who want this will want to assert that a popover/tooltip showed up on hover. If we immediately mouseout and mouseleave before they get a chance to make their assertions then this won't help them...

Maybe we should have a `hover` and an `unhover` or something?